org.munadi.Munadi.json

```diff
@@ -0,0 +1,32 @@
+{
+    "app-id": "org.munadi.Munadi",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
```

Comment:
Can you try 5.12?

Reply:
I did and it failed! Just checked, new Qt JS engine broke it, I've updated the source, testing it now ....

Reply:
@barthalion All done, tested fully working, thanks.